testing/trino-tests/src/test/java/io/trino/tests/AbstractDistributedEngineOnlyQueries.java

```diff
@@ -13,49 +13,23 @@
  */
 package io.trino.tests;
 
-import com.google.common.collect.ImmutableMap;
 import io.trino.Session;
-import io.trino.connector.MockConnectorFactory;
-import io.trino.connector.MockConnectorPlugin;
-import io.trino.testing.DistributedQueryRunner;
-import io.trino.testing.QueryRunner;
-import io.trino.tpch.TpchTable;
 import org.intellij.lang.annotations.Language;
 import org.testng.annotations.Test;
 
 import java.time.ZonedDateTime;
 import java.util.regex.Pattern;
 
-import static io.airlift.testing.Closeables.closeAllSuppress;
 import static io.trino.FeaturesConfig.JoinDistributionType.BROADCAST;
 import static io.trino.SystemSessionProperties.ENABLE_DYNAMIC_FILTERING;
-import static io.trino.plugin.memory.MemoryQueryRunner.createMemoryQueryRunner;
 import static io.trino.testing.assertions.Assert.assertEventually;
 import static io.trino.testing.sql.TestTable.randomTableSuffix;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-public class TestDistributedEngineOnlyQueries
+public abstract class AbstractDistributedEngineOnlyQueries
```

Comment:
It will result in mixed convention for multiple similar tests. I would rather do it as a followup renaming all of those (also it is painful to change rename pattern with commits on top).